CHANGES.txt

```diff
@@ -23,6 +23,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
   From Ryan Saunders
     - Fixed runtest.py failure on Windows caused by excessive escaping of the path to python.exe.
 
+  From Anatoli Babenia
```

Comment:
These entries should be ordered by contributors last name. please adjust.

Reply:
Adjusted.